app/lnd/config/index.js

```diff
@@ -2,10 +2,17 @@
 // Mac OS X: /Users/user/Library/Application Support/Lnd/tls.cert
 // Linux: ~/.lnd/tls.cert
 // Windows: TODO find out where cert is located for windows machine
-import { userInfo } from 'os'
+import { userInfo, platform } from 'os'
+
+var loc;
+if (platform() == 'linux') {
```

Comment:
as long as we are gonna add support for other OS we should add `win` here as well. Something like:

```
switch (platform()) {
  case 'darwin':
    loc = path.join(homedir, 'Library/Application\ Support/Lnd/tls.cert')
    break
  case 'linux':
    loc = path.join(homedir, '.lnd/tls.cert')
    break
  case 'win32':
    loc = path.join(homedir, 'AppData', 'Local', 'Lnd', 'tls.cert')
    break
  default:
    break
}
```